Gemfile

```diff
@@ -38,6 +38,7 @@ gem "connection_pool",                                       :require => false #
 gem "dalli",                            "~>3.2.3",           :require => false
 gem "default_value_for",                "~>4.0"
 gem "docker-api",                       "~>1.33.6",          :require => false
+gem "drb",                              "~>2.1.1",           :require => false
```

Comment:
NOTE ruby 3.1.0 has drb version 2.1.1 so that's what I used here, the latest (and what is in 3.3) is 2.2.1.

Reply:
I'm leaning towards 2.2.1.
We will be changing this in the short term anyway